Sming/Makefile-rboot.mk

```diff
@@ -153,8 +159,8 @@ TARGET		= app
 
 THIRD_PARTY_DIR = $(SMING_HOME)/third-party
 
-LIBSMING = sming
 SMING_FEATURES = none
+LIBSMING = sming
```

Comment:
"In addition, I changed some things in the rBoot-specific Makefile to make it more similar to the common one."
If you looked at the diff between `Makefile-project.mk` and `Makefile-rboot.mk` before and after this commit, instead of simply looking at the commit diff, perhaps you would understand the end result better.

By reducing the amount of differences between these two files, it becomes easier to correctly apply changes to both at the same time, and perhaps even merge parts of them into a single file one day.